metadata/modules/adnuntiusAnalyticsAdapter.json

```diff
@@ -1,4 +1,5 @@
 {
+  "NOTICE": "do not edit - this file is autogenerated by `gulp update-metadata`",
```

Comment:
Interesting case here, adnuntias bid adapter has a gvlid but not their other two adapters